cds_ils/importer/config.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# CDS-ILS is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS-ILS Importer module."""
+CDS_ILS_IMPORTER_RECORD_TAG = "//*[local-name() = 'record']"
+
+CDS_ILS_IMPORTER_PROVIDERS = {
```

Comment:
what about to put in the main `config.py` file, so that you can use it every where with `current_app.config`?
If it is not a config (overridable at some point), then it is more constants? I would change the file name.